broken symlink to .a library files

radare2 installed using ./sys/install.sh on Xubuntu 16.04.1 LTS
````
radare2 0.10.6 12730 @ linux-x86-64 git.0.10.6-221-g5800b23
commit: 5800b23e8feb06df42c7898299b4872572b7b979 build: 2016-10-26
````
symlink to ``.a`` files is broken
````
$ ls -la /usr/lib/libr_asm.a
lrwxrwxrwx 1 root root 45 Окт 26 20:00 /usr/lib/libr_asm.a -> /home/user/radare/radare2/libr/asm/libr_asm.a
$ ls -la /home/user/radare/radare2/libr/asm/
total 14216
drwxrwxr-x  5 user user     4096 Окт 26 20:00 .
drwxrwxr-x 25 user user     4096 Окт 26 19:59 ..
drwxrwxr-x 44 user user     4096 Окт 26 17:27 arch
-rw-rw-r--  1 user user    21500 Окт 26 17:27 asm.c
-rw-rw-r--  1 user user     7657 Окт 26 20:00 asm.d
-rw-rw-r--  1 user user   107680 Окт 26 20:00 asm.o
-rw-rw-r--  1 user user     1166 Окт 26 17:27 code.c
-rw-rw-r--  1 user user     7647 Окт 26 20:00 code.d
-rw-rw-r--  1 user user     9320 Окт 26 20:00 code.o
drwxrwxr-x  2 user user     4096 Окт 26 20:00 d
-rw-rw-r--  1 user user     2418 Окт 26 17:27 Jamroot
-rwxrwxr-x  1 user user 14349312 Окт 26 20:00 libr_asm.so
-rw-rw-r--  1 user user      763 Окт 26 17:27 Makefile
drwxrwxr-x  2 user user    12288 Окт 26 20:00 p
````
there is no ``libr_asm.a`` at all

when I try to build keystone, it fails:
````
$ export R_DEBUG=1
$ r2pm -i keystone
Cannot open /usr/lib/radare2/0.10.6//..
Cannot open /usr/lib/radare2/0.10.6//syscall
Cannot open /usr/lib/radare2/0.10.6//hud
Cannot open /usr/lib/radare2/0.10.6//opcodes
Cannot find symbol 'radare_plugin' in library '/usr/lib/radare2/0.10.6//bin_xtr_dyldcache.so'
Cannot find symbol 'radare_plugin' in library '/usr/lib/radare2/0.10.6//parse_z80_pseudo.so'
Cannot open /usr/lib/radare2/0.10.6//magic
Cannot open /usr/lib/radare2/0.10.6//.
Cannot open /usr/lib/radare2/0.10.6//fcnsign
PLUGIN HANDLER 0x55d79091e150 0x7efe4295d116
Cannot find symbol 'radare_plugin' in library '/usr/lib/radare2/0.10.6//asm_LM32.so'
Cannot open /home/user/.config/radare2/plugins/..
Cannot open /home/user/.config/radare2/plugins/.
Cannot open /usr/lib/radare2/0.10.6/..
Cannot open /usr/lib/radare2/0.10.6/syscall
Cannot open /usr/lib/radare2/0.10.6/hud
Cannot open /usr/lib/radare2/0.10.6/opcodes
Cannot find symbol 'radare_plugin' in library '/usr/lib/radare2/0.10.6/bin_xtr_dyldcache.so'
Cannot find symbol 'radare_plugin' in library '/usr/lib/radare2/0.10.6/parse_z80_pseudo.so'
Cannot open /usr/lib/radare2/0.10.6/magic
Cannot open /usr/lib/radare2/0.10.6/.
Cannot open /usr/lib/radare2/0.10.6/fcnsign
Dupped
Cannot find symbol 'radare_plugin' in library '/usr/lib/radare2/0.10.6/asm_LM32.so'
Cannot open directory '/usr/lib/radare2-extras/0.10.6'
Cannot open directory '/usr/lib/radare2-bindings/0.10.6'
Processing keystone ...
Already up-to-date.
Install Done For keystone
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for working directories... current
using prefix '/home/user/.config/radare2/prefix/'
checking for c compiler... gcc
checking for c++ compiler... g++
checking for libkeystone ... yes
Using PKGCONFIG: pkg-config
checking pkg-config flags for r_asm... no
This package is required

````

building keystone still fails:
````
$ r2pm -i keystone
Processing keystone ...
Already up-to-date.
Install Done For keystone
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking target system type... x86_64-unknown-linux-gnu
checking for working directories... current
using prefix '/home/user/.config/radare2/prefix/'
checking for c compiler... gcc
checking for c++ compiler... g++
checking for libkeystone ... yes
Using PKGCONFIG: pkg-config
checking pkg-config flags for r_asm... yes
creating ./config.mk
cleaning temporally files... done
rm -f *.so
make asm_arm_ks.so
make[1]: Entering directory '/home/user/.config/radare2/r2pm/git/radare2-extras/keystone'
gcc -lc++ -fPIC -shared -Wall -I/usr/include/libr -lr_asm -lr_syscall -lr_lang -lr_util -lr_parse -lr_flags -lr_cons -lr_reg -lr_anal -I/home/user/.config/radare2/prefix/include -L/home/user/.config/radare2/prefix/lib -lkeystone -o asm_arm_ks.so asm_arm_ks.c -lkeystone
/usr/bin/ld: cannot find -lc++
/usr/bin/ld: cannot find -lr_flags
collect2: error: ld returned 1 exit status
Makefile:27: recipe for target 'asm_arm_ks.so' failed
make[1]: *** [asm_arm_ks.so] Error 1
make[1]: Leaving directory '/home/user/.config/radare2/r2pm/git/radare2-extras/keystone'
Makefile:30: recipe for target 'arm' failed
make: *** [arm] Error 2
````
Issue ``/usr/bin/ld: cannot find -lc++`` is already reported:
https://github.com/radare/radare2-extras/issues/70
https://github.com/radare/radare2-extras/pull/73

issue with ``/usr/bin/ld: cannot find -lr_flags``
````
$ ls -la /usr/lib/libr_fla*
lrwxrwxrwx 1 root root 47 Окт 27 09:28 /usr/lib/libr_flag.a -> /home/user/radare/radare2/libr/flag/libr_flag.a
lrwxrwxrwx 1 root root 48 Окт 27 09:28 /usr/lib/libr_flag.so -> /home/user/radare/radare2/libr/flag/libr_flag.so
lrwxrwxrwx 1 root root 48 Окт 27 09:28 /usr/lib/libr_flag.so.0.99.9-git -> /home/user/radare/radare2/libr/flag/libr_flag.so
````
I think need change ``-lr_flags`` to ``-lr_flag`` (without ``s`` at end). I will send PR to radare2-extras

--- a/binr/radare2/radare2.c
+++ b/binr/radare2/radare2.c
@@ -31,7 +31,7 @@ static int verify_version(int show) {
 		{ "r_asm", &r_asm_version },
 		{ "r_bin", &r_bin_version },
 		{ "r_cons", &r_cons_version },
-		{ "r_flags", &r_flag_version }, // XXX inconsistency
+		{ "r_flag", &r_flag_version },
 		{ "r_core", &r_core_version },
 		{ "r_crypto", &r_crypto_version },
 		{ "r_bp", &r_bp_version },

--- a/pkgcfg/r_asm.pc.acr
+++ b/pkgcfg/r_asm.pc.acr
@@ -7,5 +7,5 @@ Name: r_asm
 Description: radare foundation libraries
 Version: @VERSION@
 Requires:
-Libs: -L${libdir} -lr_asm -lr_syscall -lr_lang -lr_util -lr_parse -lr_flags -lr_cons -lr_reg -lr_anal
+Libs: -L${libdir} -lr_asm -lr_syscall -lr_lang -lr_util -lr_parse -lr_flag -lr_cons -lr_reg -lr_anal
 Cflags: -I${includedir}/libr

--- a/pkgcfg/r_core.pc.acr
+++ b/pkgcfg/r_core.pc.acr
@@ -7,5 +7,5 @@ Name: r_core
 Description: radare foundation libraries
 Version: @VERSION@
 Requires:
-Libs: -L${libdir} -lr_core -lr_config -lr_cons -lr_io -lr_util -lr_flags -lr_asm -lr_debug -lr_hash -lr_bin -lr_lang -lr_io -lr_anal -lr_parse -lr_bp -lr_egg -lr_reg -lr_search -lr_syscall -lr_socket -lr_fs @LIBMAGIC@ -lr_crypto -lr_crypto
+Libs: -L${libdir} -lr_core -lr_config -lr_cons -lr_io -lr_util -lr_flag -lr_asm -lr_debug -lr_hash -lr_bin -lr_lang -lr_io -lr_anal -lr_parse -lr_bp -lr_egg -lr_reg -lr_search -lr_syscall -lr_socket -lr_fs @LIBMAGIC@ -lr_crypto -lr_crypto
 Cflags: -I${includedir}/libr

--- a/pkgcfg/r_debug.pc.acr
+++ b/pkgcfg/r_debug.pc.acr
@@ -7,5 +7,5 @@ Name: r_debug
 Description: radare foundation libraries
 Version: @VERSION@
 Requires:
-Libs: -L${libdir} -lr_debug -lr_reg -lr_anal -lr_bp -lr_io -lr_parse -lr_cons -lr_syscall -lr_hash -lr_flags -lr_util -lr_socket
+Libs: -L${libdir} -lr_debug -lr_reg -lr_anal -lr_bp -lr_io -lr_parse -lr_cons -lr_syscall -lr_hash -lr_flag -lr_util -lr_socket
 Cflags: -I${includedir}/libr

--- a/pkgcfg/r_flag.pc.acr
+++ b/pkgcfg/r_flag.pc.acr
@@ -7,5 +7,5 @@ Name: r_flags
 Description: radare foundation libraries
 Version: @VERSION@
 Requires:
-Libs: -L${libdir} -lr_flags -lr_util
+Libs: -L${libdir} -lr_flag -lr_util
 Cflags: -I${includedir}/libr

--- a/pkgcfg/r_parse.pc.acr
+++ b/pkgcfg/r_parse.pc.acr
@@ -7,5 +7,5 @@ Name: r_parse
 Description: radare foundation libraries
 Version: @VERSION@
 Requires:
-Libs: -L${libdir} -lr_parse -lr_flags -lr_util -lr_anal -lr_syscall -lr_reg -lr_cons
+Libs: -L${libdir} -lr_parse -lr_flag -lr_util -lr_anal -lr_syscall -lr_reg -lr_cons
 Cflags: -I${includedir}/libr

